apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -22,8 +27,11 @@ type Properties<T extends FieldValues> = {
 const Input = <T extends FieldValues>({
 	control,
 	errors,
+	iconName,
+	isDisplayedValue = false,
```

Comment:
Let's move ```isDisplayedValue``` right before placeholder parameter as function parameters with default values are usually moved to the end of parameters